Robust.Client/Audio/Midi/MidiRenderer.cs

```diff
@@ -79,7 +79,7 @@ public interface IMidiRenderer : IDisposable
         /// <summary>
         ///     Gets or sets the Time Scale of the sequencer in ticks per second. Default is 1000 for 1 tick per millisecond.
```

Comment:
```suggestion
        ///     Gets the Time Scale of the sequencer in ticks per second. Default is 1000 for 1 tick per millisecond.
```
@Zumorica like this?

Reply:
Yeah, exactly like that.